git-cliff-core/src/error.rs

```diff
@@ -111,6 +111,12 @@ pub enum Error {
 	/// Error that may occur while constructing patterns.
 	#[error("Pattern error: `{0}`")]
 	PatternError(#[from] glob::PatternError),
+	/// Error that
```

Comment:
We should complete this doc comment, maybe sth like this:

```suggestion
	/// Error that may occur when unconventional commits are found.
	/// See `require_conventional` option for more information.
```

Or anything that fits to your liking.